cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/DBSink.java

```diff
@@ -91,6 +92,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
                                                                                 getJDBCPluginId(),
                                                                                 PluginProperties.builder().build());
     Preconditions.checkArgument(jdbcDriverClass != null, "JDBC Driver class must be found.");
+    Preconditions.checkArgument(tableExists(jdbcDriverClass), "Invalid table name %s", dbSinkConfig.tableName);
```

Comment:
I believe you need to call `DBUtils.deregisterAllDrivers` as well after calling the `tableExists` in here.